bddl/activity_definitions/clean_a_backpack/problem0.bddl

```diff
@@ -12,19 +12,21 @@
         sponge.n.01_1 - sponge.n.01
         water.n.06_1 - water.n.06
         detergent.n.02_1 - detergent.n.02
+        detergent__bottle.n.01_1 - detergent__bottle.n.01
         dust.n.01_1 - dust.n.01
         agent.n.01_1 - agent.n.01
     )
     
-    (:init 
-        (ontop backpack.n.01_1 hand_towel.n.01_1) 
-        (ontop bucket.n.01_1 floor.n.01_1) 
+    (:init
+        (filled detergent__bottle.n.01_1 detergent.n.02_1)
```

Comment:
seems like the detergent might not need to be in this task - could just be the detergent bottle?